GitDiffMargin.Shim/Icons.imagemanifest

```diff
@@ -3,7 +3,7 @@
 <!-- Version: 14.0.50929.2 -->
 <ImageManifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://schemas.microsoft.com/VisualStudio/ImageManifestSchema/2014">
   <Symbols>
-    <String Name="Resources" Value="/GitDiffMargin.Shim;component/resources/icons" />
+    <String Name="Resources" Value="/GitDiffMargin.Shim;v3.11.0.0;72dfb55bb9fc8a25;component/resources/icons" />
```

Comment:
@AArnott are you aware of a way to inject this value from NBGV?

Reply:
What is that hex value? Surely the git commit id needn't be included here?

Reply:
The hex value is PublicKeyToken. The item that needs to change over time is the major/minor version.

Reply:
Well, it's easy enough to get the version within an msbuild target. For VSIXs where the version must appear, the build system allows for msbuild targets to inject values into an extension.vsixmanifest. I don't know how .imagemanifest files are treated. Are they mutated and/or copied to another directory as part of the build where a macro could be replaced during the build with the actual version number?

Reply:
➡️ The targets that detokenize vsixmanifest files operate on XML files in a way that would mostly work for imagemanifest, but they happen to validate the root element name and `<ImageManifest>` is not one of the hard-coded recognized values. I took a different approach of validating during the build that the manifest contains the expected text.

Reply:
Another way might be to remove the strong name from the assembly. Then I suspect the assembly version and token no longer have to be in the XAML.

Reply:
I'm more comfortable with the current behavior, especially now that it's build-validated and I've done a bunch of testing on it.